ivadomed/config/config_dataset.json

```diff
@@ -0,0 +1,480 @@
+{
+    "command": "train",
+    "gpu_ids": [
+        7
+    ],
+    "path_output": "pytest_output_folder",
+    "debugging": false,
+    "model_name": "unit_test",
+    "object_detection_params": {
+        "object_detection_path": null
+    },
+    "dataset_groups": [
+        {
+            "dataset_group_label": "DataSetGroup1",
+            "training": [
+                {
+                    "type": "FILES",
```

Comment:
Would it be possible to update this file to reflect the latest changes to help future reviewer?
example here:  "type" is at the dataset level and not the group level and does not match the config in [example_loader_v2_configs.py](https://github.com/ivadomed/ivadomed/pull/1201/files#diff-d1fba057563ae49e7163984ee5c4160f02af3d5bbcea113f31b9a86d06e257b3).
Thanks!

Reply:
Yeah, thanks for catching that. the example config .py was updated but I missed this example JSON. Updated now.

Reply:
Thanks for the update!
May I suggest that we use the same order everywhere for the JSON fields (in the config_dataset.json and in example_loader_v2_configs.py) , for clarity for the user.

Maybe the same order as in `example_loader_v2_configs.py` :
```
DATASET_GROUPS
    DATASET_GROUP_LABEL
    TYPE
    TRAINING
        IMAGE_GROUND_TRUTH
        MISSING_FILES_HANDLE
        EXCESSIVE_FILES_HANDLE
    VALIDATION
        IMAGE_GROUND_TRUTH
        MISSING_FILES_HANDLE
        EXCESSIVE_FILES_HANDLE
    TEST
        IMAGE_GROUND_TRUTH
        MISSING_FILES_HANDLE
        EXCESSIVE_FILES_HANDLE
    PATH_DATA
    EXPECTED_INPUT
    EXPECTED_GT
EXPECTED_INPUT
EXPECTED_GT
```

OR the following which I find kind of easier to follow:
```
DATASET_GROUPS
EXPECTED_INPUT
EXPECTED_GT
    DATASET_GROUP_LABEL
    TYPE
    PATH_DATA
    EXPECTED_INPUT
    EXPECTED_GT
    TRAINING
        IMAGE_GROUND_TRUTH
        MISSING_FILES_HANDLE
        EXCESSIVE_FILES_HANDLE
    VALIDATION
        IMAGE_GROUND_TRUTH
        MISSING_FILES_HANDLE
        EXCESSIVE_FILES_HANDLE
    TEST
        IMAGE_GROUND_TRUTH
        MISSING_FILES_HANDLE
        EXCESSIVE_FILES_HANDLE
```